README.md

```diff
@@ -106,31 +108,39 @@ shell.echo('hello world');
 
 ## Make tool
 
-A convenience script `shelljs/make` is also provided to mimic the behavior of a Unix Makefile. In this case all shell objects are global, and command line arguments will cause the script to execute only the corresponding function in the global `target` object. To avoid redundant calls, target functions are executed only once per script.
+A convenience script `shelljs/make` is also provided to mimic the behavior of a Unix Makefile.
+In this case all shell objects are global, and command line arguments will cause the script to
+execute only the corresponding function in the global `target` object. To avoid redundant calls,
+target functions are executed only once per script.
 
-Example (CoffeeScript):
+Example:
 
-```coffeescript
-require 'shelljs/make'
-
-target.all = ->
-  target.bundle()
-  target.docs()
-
-target.bundle = ->
-  cd __dirname
-  mkdir 'build'
-  cd 'lib'
-  (cat '*.js').to '../build/output.js'
-
-target.docs = ->
-  cd __dirname
-  mkdir 'docs'
-  cd 'lib'
-  for file in ls '*.js'
-    text = grep '//@', file     # extract special comments
-    text.replace '//@', ''      # remove comment tags
-    text.to 'docs/my_docs.md'
+```javascript
+require('shelljs/make');
+
+target.all = function() {
+  target.bundle();
+  target.docs();
+}
+
+target.bundle = function() {
+  cd(__dirname);
+  mkdir('build');
+  cd('lib');
+  cat('*.js').to('../build/output.js');
+}
+
+target.docs = function() {
+  cd(__dirname);
+  mkdir('docs');
+  cd('lib');
+  var files = ls('*.js');
+  for(var i = 0; i < files.length; i++) {
+    var text = grep('//@', file);     // extract special comments
+    text = text.replace('//@', '');   // remove comment tags
+    text.to('docs/my_docs.md');
+  }
+};
```

Comment:
I haven't tested this example, but if you think it works, it should be good enough for me.